src/components/containers/popover/PopoverLink.js

```diff
@@ -60,6 +61,7 @@ export class PopoverLink extends React.Component {
             styleType={buttonStyle}
             isLinkButton={isLinkButton}
             suppressUnderline={suppressUnderline}
+            aria-label={this.props.ariaLabel && ariaLabel}
```

Comment:
I don't understand the reasoning behind this. They're both the same thing, so the `&&` operator seems a bit extraneous